controllers/default.py

```diff
@@ -26,6 +26,7 @@
 import requests
 import utilities
 
+
```

Comment:
I would request a separate PR for these changes please (along with the changes for other controllers).